cli/util_internal_test.go

```diff
@@ -41,6 +41,34 @@ func TestDurationDisplay(t *testing.T) {
 	}
 }
 
+func TestExtendedParseDuration(t *testing.T) {
+	t.Parallel()
+	for _, testCase := range []struct {
+		Duration   string
+		Expected   time.Duration
+		ExpectedOk bool
+	}{
+		{"1d", 24 * time.Hour, true},
+		{"1y", 365 * 24 * time.Hour, true},
+		{"10s", 10 * time.Second, true},
+		{"1m", 1 * time.Minute, true},
+		{"20h", 20 * time.Hour, true},
+		{"10y10d10s", 0, false},
```

Comment:
I think users will expect durations like 1y1d to work here. It shouldn't be a huge lift to enable this!

Reply:
Hey @johnstcn I tried to play with time.ParseDuration golang function to mimic the behavior as much as possible - and changed to indeed modify what you've raised.